service/src/main/java/bio/terra/cbas/service/RunSetsService.java

```diff
@@ -468,9 +473,7 @@ private SubmitRunSetResponse submitRunSet(
         cromwellSubmitRunsSample,
         "cromwell_submit_runs_timer",
         RunSet.RUN_SET_ID_COL,
-        runSet.runSetId().toString(),
-        "runStateResponseList",
-        runStateResponseList.toString());
```

Comment:
I removed this because this is sending the whole RunSetResponse to custom dimensions which is not very useful and can get very large for larger submissions.